plugin/trino-vertica/src/main/java/io/trino/plugin/vertica/VerticaTableStatisticsReader.java

```diff
@@ -0,0 +1,488 @@
+        {
+            return distinct.getValue();
+        }
+
+        public NullableValue getMinValue()
+        {
+            return minValue;
+        }
+
+        public NullableValue getMaxValue()
+        {
+            return maxValue;
+        }
+
+        public Histogram getHistogram()
+        {
+            return histogram;
+        }
+
+        @Override
+        public String toString()
+        {
+            return toStringHelper(this)
+                    .add("distinct", distinct)
+                    .add("minValue", minValue)
+                    .add("maxValue", maxValue)
+                    .add("histogram", histogram)
+                    .toString();
+        }
+    }
+
+    public static class Histogram
+    {
+        @JacksonXmlElementWrapper(useWrapping = false)
+        private final List<Histogram.Category> category;
+
+        @JsonCreator
+        public Histogram(@JacksonXmlProperty(localName = "category") List<Category> category)
+        {
+            this.category = category;
+        }
+
+        public List<Histogram.Category> getCategories()
+        {
+            return category;
+        }
+
+        public static class Category
+        {
+            private final NullableValue bound;
+            private final NumericValue count;
+
+            @JsonCreator
+            public Category(
+                    @JacksonXmlProperty(localName = "bound") NullableValue bound,
+                    @JacksonXmlProperty(localName = "count") NumericValue count)
+            {
+                this.bound = bound;
+                this.count = count;
+            }
+
+            public NullableValue getBound()
+            {
+                return bound;
+            }
+
+            public NumericValue getCount()
+            {
+                return count;
+            }
+        }
+    }
+
+    public static class NullableValue
+    {
+        private final String value;
+        private final String nullValue;
+
+        @JsonCreator
+        public NullableValue(String value)
+        {
+            this(null, value);
+        }
+
+        @JsonCreator
+        public NullableValue(
+                @JacksonXmlProperty(localName = "nullValue", isAttribute = true) String nullValue,
+                @JacksonXmlProperty(localName = " ") String value)
+        {
+            this.value = value;
+            this.nullValue = nullValue;
+        }
+
+        public Optional<String> getValue()
+        {
+            return Optional.ofNullable(value);
+        }
+
+        public boolean isNull()
+        {
+            if (nullValue == null) {
+                return false;
+            }
+            return nullValue.equals("true");
+        }
+
+        public boolean isNan()
+        {
+            if (value == null) {
+                return false;
+            }
+            return value.equals("-nan") || value.equals("nan");
+        }
+    }
+
+    public static class NumericValue
```

Comment:
a class for long looks like an overkill

Reply:
This class is used to retrieve `value` attribute from:
```xml
<distinct value="25"/>
```

```xml
<count value="1"/>
```
It it possible to get the value without this class?